server/routes/calculationRoutes.ts

```diff
@@ -130,6 +130,12 @@ export default class CalculationRoutes {
     const breakdown = await this.calculateReleaseDatesService.getBreakdown(calculationRequestId, token)
     const sentencesAndOffences = await this.viewReleaseDatesService.getSentencesAndOffences(calculationRequestId, token)
     const hasNone = 'None' in releaseDates.dates
+    let approvedDates
+    if (releaseDates.approvedDates) {
+      approvedDates = this.indexApprovedDates(releaseDates.approvedDates)
+    } else {
+      approvedDates = null
+    }
```

Comment:
preferable to use const if possible.. so above can be changed to something like :
`const approvedDates = releaseDates.approvedDates ? this.indexApprovedDates(releaseDates.approvedDates) : null`